README.md

```diff
@@ -49,10 +49,10 @@ After a doctor completes this process, this platform provides two main features.
 2) Secondly, for a given patient, all suitable matches from all the registered hospitals are displayed. These suitable matches are ranked from best to worst.
 
 ## Documentation
-[First timers Guide](./docs/newContributors.md): **New Contributors** follow this to create **your first Pull Request**.  
-[Contributing Guide](./docs/CONTRIBUTING.md): Please follow the rules mention here while contributing to this project.  
+[First timers Guide](./docs/newContributors.md): **New Contributors** follow this to create **your first Pull Request**. 
+[Contributing Guide](./docs/CONTRIBUTING.md): Please follow the rules mention here while contributing to this project.
 [Build documentation](./docs/build.md): Follow this to setup the project in your local machine.  
-[Scoring Functions](https://github.com/siv2r/kidney-exchange/projects/1): Parameters used for find the match score.  
+[Scoring Functions](https://github.com/siv2r/kidney-exchange/projects/1): Parameters used for find the match score.
 [FAQs](./docs/FAQs.md): Frequently asked questions  
```

Comment:
These changes mess up the alignment when the readme file is rendered. Can you undo all the changes made in this file?

Reply:
Really sorry, I am kinda new to this, didn't mean to mess it up.

Reply:
No worries